.github/workflows/wheels.yml

```diff
@@ -109,7 +109,7 @@ jobs:
       - name: setup rtools for 32-bit
         run: |
           echo "PLAT=i686" >> $env:GITHUB_ENV
```

Comment:
I think you need a  `choco install rtools -y --no-progress --force --version=4.0.0.20220206` line here. Otherwise you'll get rtools43 by default.